src/diffusers/models/cross_attention.py

```diff
@@ -95,14 +97,18 @@ def __init__(
             self.add_v_proj = nn.Linear(added_kv_proj_dim, cross_attention_dim)
 
         self.to_out = nn.ModuleList([])
-        self.to_out.append(nn.Linear(inner_dim, query_dim))
+        self.to_out.append(nn.Linear(inner_dim, query_dim, bias=out_bias))
         self.to_out.append(nn.Dropout(dropout))
 
         # set attention processor
-        # We use the AttnProcessor2_0 by default when torch2.x is used which uses
+        # We use the AttnProcessor2_0 by default when torch 2.x is used which uses
         # torch.nn.functional.scaled_dot_product_attention for native Flash/memory_efficient_attention
+        # but only if it has the `scale` argument
         if processor is None:
-            processor = AttnProcessor2_0() if hasattr(F, "scaled_dot_product_attention") else CrossAttnProcessor()
+            if torch.torch_version.TorchVersion(torch.__version__) >= (2, 1, 0):
```

Comment:
Let's revert this, we don't need 2.1, 2.0 is enough and I think the logic before was good

Reply:
right but then the `scaled_dot_product_attention` in 2.0 has no `scale` which is what i would need... but yes i can deal with that in the pipeline?

Reply:
Ah I see, ok I think it's fine if Torch 2.0 doesn't work yet for the spectrogram model. Let's maybe just advertise it with the previous PyTorch version and see if the community tries it out on Pytorch 2.0

Reply:
ok cool! reverting... i can deal with it or i can also check if `attn.scale == 1` and not do this... which is only for spectrogram for now?